octodns_powerdns/__init__.py

```diff
@@ -69,7 +70,7 @@ def __init__(
         api_key,
         port=8081,
         scheme="http",
-        ssl_verify=True,
+        ssl_verify=False,
```

Comment:
I too would expect many, if not most internal cases to use `False` here, but this change would be non-backwards compatible since people who were previously wanting/expecting verification and getting it via the default would no longer get that verification unless they added the parameter explicitly. Due to that it's a no-go except outside of a major release bump and would need to be "deprecated"/warned about for a major release before that.

> We would not want to set the default to False however, it is better to do security by default.

Agreed.